### DISCUSSION TITLE
LVFS HSI Applications/Collaboration

### DISCUSSION THREAD

**Discussion by johnloucaides** [chipsec/chipsec#1400] [Ideas]
LVFS/fwupd has been working on a Host Security ID specification (HSI).
https://people.freedesktop.org/~hughsient/temp/html/hsi.html
https://fossies.org/linux/fwupd/docs/hsi.md

Many of the checks are included in chipsec. The intent of this discussion is to figure out what, if any, collaboration or applications we should start doing with LVFS on this effort.

**Comment by hughsie** [comment]
Perhaps interestingly for some, we're also plumbing HSI all the way up the desktop stack, e.g. in Plymouth for SecureBoot=false/invalid, in GDM for PCR0 changes (i.e. not _unlock before aware_) and making this available in the control center. e.g. https://gitlab.gnome.org/Teams/Design/settings-mockups/-/blob/master/device-security/device-security.png -- from a server point of view HSI is now a first-class citizen in Red Hat Insights too.
